src/Handler/Search.hs

```diff
@@ -59,7 +59,7 @@ routeResult SearchResult{..} =
       ( PackageVersionModuleDocsR ppkgName pversion modName
       , Nothing
       )
-    DeclarationResult typeOrValue modName declTitle typeText ->
+    DeclarationResult typeOrValue modName declTitle _typeText ->
```

Comment:
If it turns out we want to use it in the future then we can always change it back to `typeText` without the leading underscore (and in that case I'd rather we did do that than using the leading underscore version).